Angular/src/app/tenants/tenants/viewTenants/view-tenants/createTenantModal/create-tenant-modal/create-tenant-modal.component.html

```diff
@@ -72,13 +124,24 @@ <h2>Add New Tenant</h2>
           [(ngModel)]="tenantModel.companyNumber"
           name="companyNumber"
           required
+          maxlength="10"
+          pattern="[0-9]*"
         />
+        <!--<div class="if" *ngIf="tenantForm.get('companyNumber')?.hasError('required') && tenantForm.get('companyNumber')?.touched">
+          {{ errorMessages.required }}
+        </div>
+        <div class="if" *ngIf="tenantForm.get('companyNumber')?.hasError('maxlength') && tenantForm.get('companyNumber')?.touched">
+          {{ errorMessages.maxlength }}
+        </div>
+        <div *ngIf="tenantForm.get('companyNumber')?.hasError('pattern') && tenantForm.get('companyNumber')?.touched">
+          {{ errorMessages.pattern }}
+        </div>-->
       </mat-form-field>
-            <div class="modal-actions">
-
-        <button mat-raised-button type="submit" color="primary" (click)="closeModal()">
```

Comment:
This click can't be CreateTenant()
It breaks the code and adds tenant multiple times as its already a CreateTenant() submission form

Reply:
Oh no

Reply:
Don't merge this branch - I am closing it